homeassistant/components/home_connect/switch.py

```diff
@@ -207,12 +264,15 @@ async def async_turn_on(self, **kwargs: Any) -> None:
 
     async def async_turn_off(self, **kwargs: Any) -> None:
         """Switch the device off."""
+        if self.power_off_state is None:
+            _LOGGER.debug("This appliance type does not support turning off")
+            return
```

Comment:
Do these devices still have a state?
Not sure if we prefer a sensor + button or a switch that can only do one action (assuming they can still turn on)

Reply:
I think so, from [API Docs](https://api-docs.home-connect.com/settings/#power-state):
![image](https://github.com/user-attachments/assets/ebd3cc89-57dc-482b-ac2e-e9a5208b330b)